crypto/cryptlib.c

```diff
@@ -272,6 +292,27 @@ void OPENSSL_showfatal(const char *fmta, ...)
     va_end(ap);
 
 # if defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0333
+#  ifdef OPENSSL_SYS_WIN_CORE
+    /* ONECORE is always NONGUI and NT >= 0x0601 */
+    if (SUCCEEDED(TraceLoggingRegister(g_hProvider))) {
+        TraceLoggingWrite(g_hProvider, "ErrorLog",
+                            TraceLoggingLevel(EVENTLOG_ERROR_TYPE),
+                            TraceLoggingWideString(buf, "Message"));
+        /* OneCore fails to compile if UNICODE is not defined, use UNICODE. */
+    }
+    else {
```

Comment:
I'm sorry, but style guide is quite explicit about this, it's `} else {` on one line, not two.

Reply:
it's ok, more readable now

Reply:
On *side* note [and side note means that no action is expected], one can wonder if it's appropriate to OutputDebugString not as `else` to failure to add a log entry, but always. Indeed, if you have a debugger attached you probably would appreciate the message there, as opposite to having to look for it elsewhere. Once again, this is a side note.

Reply:
Just wanted to use it same way as it is used by standard configuration, to be more consistent.
Imho it can be good idea to present debug log regardless event was thrown properly or not.